docs/reference/catalogs.md

```diff
@@ -1 +1,24 @@
 # Catalogs
+
+Catalogs are libraries of operators. Catalogs are what extends Shift's capabilities, as it provides the specialized nodes for different areas of development and automatization. 
+
+Shift comes with six native catalogs that contain the necessary nodes to start building a workflow:
+
+- **Constants catalog**: Provides operators to work with standard Python type objects, such as booleans or strings.
+- **Standard catalog**: Provides operators to work with standard Python operations. It provides logic to perform [loop iterations](../reference/nodes/iteration.md) and create [scriptable Python nodes](../reference/nodes/python_script.md). It also includes operators to read/write JSON files, operate with lists, strings, dictionaries and more.
+- **FileSystem catalog**: Provides operators to work with the OS filesystem and environment variables. It includes operators to perform file & directory creation and manipulation and more.
+- **Regex catalog**: Provides operators to work with regular expressions and functions.
+- **CreativeSoftware catalog**: Provides operators to work with Digital Content Creation softwares. It includes operators for scene management, geometry handling, node graph manipulation and more, across popular DCC tools like Maya, Houdini, Blender, and Nuke.
+- **Workflow catalog**: Provides operators to manage and nest Shift workflows. It includes operators to input and output data, store [variables](../reference/nodes/variables.md) and execute [sub-workflows](../reference/nodes/workflow.md).
+
+>[!NOTE]
+> Shift also comes with a native catalog for Deadline. For more information about Shift's integration in Deadline and how to use this catalog, please refer to the [Deadline Integration](../integration_resources/deadline.md) documentation.

```

Comment:
```suggestion
> Shift also comes with a native catalog of nodes that support launching Shift workflows as Deadline jobs. For more information about Shift's integration in Deadline and how to use this catalog, please refer to the [Deadline Integration](../integration_resources/deadline.md) documentation.

```

Reply:
I'm not sure if it's a good idea to be that specific, since in the future we might add new nodes that might be used for other actions and we will have to remember to update the docstrings.

Reply:
It's not really a catalog, Right now it is just one node that can submit a Shift workflow to Deadline. In the future we will need to extend this catalog with more nodes

Reply:
ok then. Resolving